<?xml version="1.0"?>
<launch>
  <node name="grasp_detector" pkg="grasp_detection" type="grasp_detection_node" cwd="node">
  
  <param name="depth_topsub" value="/camera/depth/image_raw"/>
  <param name="rgb_topsub" value="/camera/color/image_raw"/>
  <param name="cloud_topsub" value="/asus/depth/points"/>
  <param name="cloud_toppub" value="/grasp_detection/depth/mypoints"/>

  <param name="saved_rgb_dir" value="$(find grasp_detection)/data/rgb.png"/>
  <param name="saved_depth_dir" value="$(find grasp_detection)/data/depth.png"/>
  <param name="saved_points_dir" value="$(find grasp_detection)/data/points.ply"/>
  
  <param name="optical_frame_id" value="asus_rgb_optical_frame"/> 
  <param name="grasp_frame_id" value="world"/>
  <param name="table_height" value="0.7"/>
  <param name="image_save" value="true"/>

  <param name="fx" value="581.8"/>
  <param name="fy" value="581.5"/>
  <param name="cx" value="319.0"/>
  <param name="cy" value="237.3"/>
  <param name="depth_factor" value="1000"/>

  <param name="grasp_path" value="$(find grasp_detection)/data/grasps.txt"/>
  <param name="quality_thresh" value="0.0"/>
  <param name="dst_gripper2object" value="0.1"/>

  </node>
  <!-- <node name="rviz" pkg="rviz" type="rviz" args="-d $(find grasp_detection)/launch/grasp_detection.rviz"/> -->
</launch>
